Havit.Blazor.Documentation/Pages/Components/HxInputTagsDoc/HxInputTags_Documentation.razor

```diff
@@ -58,8 +58,8 @@
     </MainContent>
 
     <CssVariables>
-        <ComponentApiDocCssVariable Name="--hx-input-tags-tag-margin" Default=".25rem .25rem .25rem 0">
-            Margin of tags.
+        <ComponentApiDocCssVariable Name="--hx-input-tags-tag-gap" Default=".25rem .25rem .25rem 0">
```

Comment:
@rqx110 the default value should be `.25rem` instead of `.25rem .25rem .25rem 0`

Reply:
Well, the main purpose of this PR is to modify the default spacing between tags. Do you mean that developers need to set this value themselves?